### PR TITLE
new ai lawset: repurposed syndOS laws for station use

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds a new lawset for ai, "Command Control". Its similarly worded to the syndicate lawset, but instead targets command staff, the logic being that command is the most important for running the station, thus protect them from rogue crew, and the command staff can appoint others to be "important" to the ai in case of emergency.

Also you can use the circuitboard in hand to replace the command staff with anything you want, from a certain department to people wearing masks

## Why It's Good For The Game
Adds a bit more chaos, makes the command staff more protected, modifying it with the circuitboard can lead to zany situations like rouge medical

## Changelog

:cl:
add: new ai lawset: command control
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
how is this different from just using asimov inhand to change target to "member of x group"?

**Comment by MrStonedOne** [comment]
it can be made a random lawsuit via the config and/or station traits.

**Comment by etherware-novice** [comment]
lawsuit

**Review by MrMelbert — CHANGES_REQUESTED** [review]
I think the lawset is unnecessary when we have our asimov boards, but these should be addressed at the very least

File: code/game/objects/items/AI_modules/full_lawsets.dm [hunk 1]
```diff
@@ -161,3 +162,19 @@
...
+/obj/item/ai_module/core/full/group_protect/attack_self(mob/user)
+	var/target_name = tgui_input_text(user, "Enter a new group that Command Control is concerned with.", "Command Control", subject, MAX_NAME_LEN)
+	if(!target_name)
+		return
+	laws.Cut()
+	var/datum/ai_laws/group_protect/lawset = new
+	subject = target_name
+	for (var/law in lawset.inherent)
+		laws += replacetext(replacetext(law, "command staff", subject), "command", subject)
+	..()
```
[the remark this answers is not in the document]
Comment (etherware-novice) [reply code/game/objects/items/AI_modules/full_lawsets.dm:180]: asimov attack_self has the same issue, i just copied from that

**Comment by etherware-novice** [comment]
@MrMelbert Hello, can you review this pr again?

**Comment by etherware-novice** [comment]
please rereview

**Comment by Mothblocks** [comment]
It does not look like any maintainer is interested in this PR and we have too many open PRs

**Pull request closed without merging**